keps/sig-docs/20191020-doc-policies-for-third-party-content.md

```diff
@@ -0,0 +1,169 @@
+
+### Implementation Details/Notes/Constraints
+
+While SIG Docs approvers may occasionally need to consider intent, any policies produced by this KEP must be clear enough to minimize cases where intent needs to be considered.
+
+(At the time of writing) Kubernetes requires external software to implement Cluster Networking, Ingress, Persistent Storage, and Logging. Hyperlinking to vendor software and documentation _is_ allowed; creating “how to use” content is not.
+
+Examples of allowed content:
+ - Cluster Networking
+   - https://kubernetes.io/docs/concepts/cluster-administration/networking/
+   - https://kubernetes.io/docs/concepts/cluster-administration/addons/
+ - Ingress Controllers
+   - https://kubernetes.io/docs/concepts/services-networking/ingress-controllers/#additional-controllers
+ - Persistent Volumes
+   - https://kubernetes.io/docs/concepts/storage/persistent-volumes/#expanding-persistent-volumes-claims
+
+Examples of content that would not be allowed:
+ - [Install a Network Policy Provider](https://kubernetes.io/docs/tasks/administer-cluster/network-policy-provider/) and child pages: how to use Calico, Cilium, Kube-router, Romana, and Weave Net for NetworkPolicy
+ - [Audit](https://kubernetes.io/docs/tasks/debug-application-cluster/audit/)
+ - [Use fluentd to collect and distribute audit events from log file](https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#use-fluentd-to-collect-and-distribute-audit-events-from-log-file) (dual-sourced)
+ - [Use logstash to collect and distribute audit events from webhook backend](https://kubernetes.io/docs/tasks/debug-application-cluster/audit/#use-logstash-to-collect-and-distribute-audit-events-from-webhook-backend) (vendor-specific content)
+ - [Auditing with Falco](https://kubernetes.io/docs/tasks/debug-application-cluster/falco/) (dual-sourced)
+ - [Events in Stackdriver](https://kubernetes.io/docs/tasks/debug-application-cluster/events-stackdriver/) (vendor-specific content)
+ - [Logging Using Elasticsearch and Kibana](https://kubernetes.io/docs/tasks/debug-application-cluster/logging-elasticsearch-kibana/) (vendor-specific content)
+ - [Logging using Stackdriver](https://kubernetes.io/docs/tasks/debug-application-cluster/logging-stackdriver/) (vendor-specific content)
+
+### Risks and Mitigations
+
+- Rejecting a change that is ultimately deemed acceptable.
+- Accepting a change that is ultimately judged to be unacceptable vendor promotion.
+- It may not always be possible to achieve consensus on whether specific third-party content should be included.
+  This may lead to resentment/resistance to future third party content.
+  *Mitigation*: consider including folks that also work outside the relevant SIG or subproject.
+
+## Design Details
+
+### Graduation Criteria
+
+**Note:** *this KEP does not target any release*
+
+TBD
+
+## Drawbacks
+
+_Why should this KEP _not_ be implemented._
+
+TBD
```

Comment:
Thoughts on this, anyone?

Reply:
I don't mean this to be snarky at all, but SIG-Docs currently makes the call as to what goes into the docs and what stays out of the docs. Arguably, it's in SIG-Docs's charter to set these standards themselves without the need of a KEP.

Reply:
I think these policies should be transparent, regardless of whether they live in a kep or in the SIG-Docs charter.

Reply:
Thanks for the feedback. This KEP is at least partly there to document the _opportunity_ for discussion and to make things visible to people outside of SIG Docs.